index.html

```diff
@@ -1,219 +1,336 @@
-                  <div class="social">
-                     <a href="#" title="Github"><i class="fab fa-github"></i></a>
-                     <a href="#" title="Twitter"><i class="fab fa-twitter"></i></a>
-                     <a href="#" title="Linkedin"><i class="fab fa-linkedin"></i></a>
-                     <a href="#" title="Portfolio"><i class="fas fa-link"></i></a>
-                  </div>
-               </div>
-            </a>
-
-            <a href="templates/moderator-card.html" class="card">
-               <img src="https://avatars.githubusercontent.com/u/53310847?v=4" alt="Kaiwalyakoparkar">
-               <div>
-                  <h3>Kaiwalyakoparkar</h3>
-                  <h4>Community Developer</h4>
-               </div>
-            </a>
-
-            <a href="templates/moderator-card.html" class="card">
-               <img src="https://avatars.githubusercontent.com/u/59364507?v=4" alt="Galahad42">
-               <div>
-                  <h3>Galahad42</h3>
-                  <h4>Community Developer</h4>
-               </div>
-            </a>
-            
-            <a href="templates/moderator-card.html" class="card">
-               <img src="https://avatars.githubusercontent.com/u/61582763?v=4" alt="Akshu-on-github">
-               <div>
-                  <h3>Akshu-on-github</h3>
-                  <h4>Community Developer</h4>
-               </div>
-            </a>
-
-            <a href="templates/moderator-card.html" class="card">
-               <img src="https://avatars.githubusercontent.com/u/65516863?v=4" alt="LandonLloyd">
-               <div>
-                  <h3>LandonLloyd</h3>
-                  <h4>Community Developer</h4>
-               </div>
-            </a>
-
-            <a href="templates/moderator-card.html" class="card">
-               <img src="https://avatars.githubusercontent.com/u/81645563?v=4" alt="DennaGrey">
-               <div>
-                  <h3>DennaGrey</h3>
-                  <h4>Community Support</h4>
-               </div>
-            </a>
-
-            <a href="templates/moderator-card.html" class="card">
-               <img src="https://avatars.githubusercontent.com/u/81779470?v=4" alt="Whisperdawn">
-               <div>
-                  <h3>Whisperdawn</h3>
-                  <h4>Community Support</h4>
-               </div>
-            </a>
-         </div>
+        <div class="moderators-header">
+          <h2>Community Team</h2>
+          <p>
+            These folks are the members of our core community team, who
+            volunteer their time to make sure everything runs smoothly!
+          </p>
+        </div>
+        <div class="moderators-header">
+          <h3>Community Managers</h3>
+        </div>
+        <div class="moderators">
+          <a href="templates/moderator-card.html" class="card">
+            <img src="https://github.com/nhcarrigan.png" alt="Nhcarrigan" />
+            <div>
+              <h3>Nhcarrigan</h3>
```

Comment:
Maybe use your real-life name instead? 🤷‍♂️
```suggestion
              <h3>Nicholas Carrigan</h3>
```

Reply:
Nah, I'm more recognisable by my username for sure lol